provisioner/aws.go

```diff
@@ -410,11 +410,7 @@ func (a *awsAdapter) CreateOrUpdateEtcdStack(parentCtx context.Context, stackNam
 	}
 
 	resp, err := a.cloudformationClient.DescribeStacks(describeParams)
-	if err != nil {
-		return err
-	}
-
-	if len(resp.Stacks) == 1 {
+	if err ==nil && len(resp.Stacks) == 1 {
```

Comment:
Please fix the formatting!

Reply:
I'd also add some comment about why we're ignoring the error (not a big deal and CF API doesn't report non-existent stacks in an obvious way).

Reply:
I raised the same question in person. Please comment it why ignoring an error 🙂.